src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HPackEncoder.cs

```diff
@@ -318,7 +318,7 @@ public static bool EncodeLiteralHeaderFieldWithoutIndexingNewName(string name, S
             // | Value String (Length octets)  |
             // +-------------------------------+
 
-            if ((uint)destination.Length >= 2)
+            if ((uint)destination.Length != 0)
```

Comment:
Why did this change?  Is it possible for nameLength to be 0 successfully?

Reply:
smaller instruction size and code below it already checks for length.